### ISSUE TITLE
Bug in tweakreg/imalign

### ISSUE THREAD

**Issue by hbushouse** [spacetelescope/jwst#2804]
#2796 introduced a bug in imalign.py, which was caught by the nightly regression tests:
```
            if len(group.catalog) < minobj:
                for i in group:
                    log.warning("Not enough sources to align image {:s}. "
                                "This image will not be aligned."
                                .format(i.meta['image_model'].meta.filename))
                    i.meta['image_model'].meta.cal_step.tweakreg = "SKIPPED"
                    i.meta['skip_alignment'] = True
                skipped_imcat.append(group)
    
            else:
>               i.meta['skip_alignment'] = False
E               UnboundLocalError: local variable 'i' referenced before assignment

../../jwst/tweakreg/imalign.py:198: UnboundLocalError
```
In the new code, when len(group.catalog) is >= minobj, the variable "i" is not set when executing the "else" half of the if/else block.

**Comment by mcara** [comment]
Yeah... I'll fix it in a moment. Sorry

**Comment by mcara** [comment]
@hbushouse This should be fixed via https://github.com/spacetelescope/jwst/pull/2805 Please merge that PR at your earliest convenience.

**Comment by hbushouse** [comment]
Fixed by #2805
